domains/iviswanathreddy.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Personal portfolio written in Ruby and HTML",
+    "repo": "https://github.com/iviswanathreddy/iviswanathreddy.github.io.git",
+    "owner": {
+        "username": "iviswanathreddy",
+        "email": "i.viswanath666@gmail.com",
+    },
+    "record": {
+        "CNAME": "https://iviswanathreddy.github.io"
```

CNAME cannot have https://
```suggestion
        "CNAME": "iviswanathreddy.github.io"
```